os/drivers/lcd/st7789.c

```diff
@@ -547,37 +547,6 @@ static void st7789_rdram(FAR struct st7789_dev_s *dev, FAR uint8_t *buff, size_t
 	SPI_SELECT(dev->spi, SPIDEV_DISPLAY, false);
 }
 
-/****************************************************************************
```

Comment:
Let's split commit for this. And could you write more detail in commit description for this?